### ISSUE TITLE
bn层 lr_mult：0

### ISSUE THREAD

**Issue by yu1206** [senlinuc/caffe_ocr#145]
训练时 ，bn层 lr_mult 为什么是0

layer {
  bottom: "conv1"
    top: "conv1"
    name: "bn_conv1"
    type: "BatchNorm"
    batch_norm_param {
    use_global_stats: false
      }
    param { lr_mult: 0 }
    param { lr_mult: 0 }
    param { lr_mult: 0 }
}
